jenkins/common.groovy

```diff
@@ -74,15 +74,6 @@ def slackIntegrationNotify() {
 
 def setup() {
   stage("Setup") {
-
-    dir("vagov-content") {
```

Comment:
This is unrelated to this PR, but this reminded me that we can probably remove `vagov-content` from the `install-repos.sh` script.

Reply:
I think CMS team uses that, I would check with them first